docs/plugins/google-cloud.md

```diff
@@ -1,72 +1,116 @@
 # Google Cloud plugin
```

Comment:
Meta question: which plug-in should folks to use? google-cloud or firebase? In this revamp, should we make our preference more apparent and opinionated?